hare/preroundtracker.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 type PreRoundTracker struct {
-	preRound  map[string]struct{} // maps PubKey->Pre-Round msg (unique)
-	tracker   *RefCountTracker    // keeps track of seen values
-	threshold uint32              // the threshold to prove a single value
+	preRound  map[string]*Set  // maps PubKey->Pre-Round msg (unique)
```

Comment:
I don't understand the comment - what is unique? and should it be Set instead of "Pre-Round msg"?

Reply:
Also (general comment) - please add tests to test these changes. It is suspicious that you a major logical change in the PreroundTracker without needing to change any of the test code, please make sure that this struct is well tested.